src/views/listview/list-view-directive.js

```diff
@@ -620,7 +622,11 @@ angular.module('patternfly.views').directive('pfListView', function ($timeout, $
           }
         }
         if (scope.config.onClick) {
-          scope.config.onClick(item, e);
+          clickReturnValue = scope.config.onClick(item, e);
+          enableRowExpansion = scope.config && scope.config.useExpandingRows && item && !item.disableRowExpansion;
+          if (typeof clickReturnValue === 'undefined' && enableRowExpansion) {
+            scope.toggleItemExpansion(item);
+          }
```

Comment:
If there is no onClick function in the config, then expansion should happen.

ie:

if (!angular.isFunction(scope.config.onClick) || scope.config.onClick(item, e) !== false) {
  ...
}